homeassistant/components/ntfy/notify.py

```diff
@@ -28,6 +35,37 @@
 PARALLEL_UPDATES = 0
 
 
+SERVICE_PUBLISH = "publish"
+ATTR_ATTACH = "attach"
+ATTR_CALL = "call"
+ATTR_CLICK = "click"
+ATTR_DELAY = "delay"
+ATTR_EMAIL = "email"
+ATTR_ICON = "icon"
+ATTR_MARKDOWN = "markdown"
+ATTR_PRIORITY = "priority"
+ATTR_TAGS = "tags"
+
+SERVICE_PUBLISH_SCHEMA = cv.make_entity_service_schema(
+    {
+        vol.Optional(ATTR_TITLE): cv.string,
+        vol.Optional(ATTR_MESSAGE): cv.string,
+        vol.Optional(ATTR_MARKDOWN): cv.boolean,
+        vol.Optional(ATTR_TAGS): vol.All(cv.ensure_list, [str]),
+        vol.Optional(ATTR_PRIORITY): vol.All(vol.Coerce(int), vol.Range(1, 5)),
+        vol.Optional(ATTR_CLICK): vol.All(vol.Url(), vol.Coerce(URL)),
+        vol.Optional(ATTR_DELAY): vol.All(
+            cv.time_period,
+            vol.Range(min=timedelta(seconds=10), max=timedelta(days=3)),
+        ),
+        vol.Optional(ATTR_ATTACH): vol.All(vol.Url(), vol.Coerce(URL)),
+        vol.Optional(ATTR_EMAIL): vol.Email(),
+        vol.Optional(ATTR_CALL): cv.string,
```

Comment:
As I understand it, these two are not allowed together with a delay, I think we should check and reject that.